03 - Showcases & Templates/Vaults/🗂️ Vaults.md

```diff
@@ -9,6 +9,8 @@ tags:
 
 #placeholder/description
 
+[Blue topaz themes example vault for Obsidian](https://github.com/cumany/Blue-topaz-examples)
```

Comment:
Thanks!

**Thinking about location**

Is this linked vault showing how to structure a vault, or is it showcasing a theme?

I ask because this folder seems to be about sharing vault structure, rather than showcasing themes. 

If it is to showcase the theme, I wonder whether it might be worth adding a new folder "Theme Showcases"?

**Thinking about format**

If the new info does stay in this folder...

Reading the "Contributing" notes on this page, it seems the intention was to have a page-per-vault.

There is a template called "Vault Showcase" - and the existing notes match that format.

I'd suggest making a minimal page for this vault, and also in that page, adding a link to the theme's page, as the backlink will be useful.

Reply:
I get what you mean, but I wouldn't be able to do that since I am not the creator. 

Also I think until we have more Hub content, I think adding links alone is fine. Having make "add a description" a condition for every addition of a link would be a bit of a roadblock on the path of having more content in the hub. Probably also rather a discussion for the hub in general than this PR though.

Reply:
OK fine... Definitely happy to go with your more knowledge of the Hub, of course...

FWIW, I think that having a bare link with no heading and not even in a list looks quite confusing, but I definitely agree that it's much better to have the link than not.

I'll approve the change - please feel free to go ahead and merge.

Reply:
oh, you are right, I added it to a list and subheading.

Nonetheless, you got a point though, having a mix of things with a Hub Page and things without one isn't really optimal. Requiring every piece of content to its own page also isn't ideal since it increases the threshold for contributions. Probably we should discuss this in Discord at some point. Otherwise merging this PR for now.

Reply:
Brilliant - thanks!